HybridCamLib/src/cam/CamView+Setup.swift

```diff
@@ -53,7 +53,7 @@ extension CamView {
          captureSession.addOutput(videoOutput)
          guard let connection = videoOutput.connection(with: .video) else { throw SetupError.unableToCreateVideoConnection }
          if connection.isVideoStabilizationSupported {/*Causes a glitch on enabled*/
-            connection.preferredVideoStabilizationMode = .auto
+            connection.preferredVideoStabilizationMode = .standard //some videos look quite shaky to me sometimes with .auto
```

Comment:
Maybe make a note about the difference of .auto and .standard in the method comment section

Reply:
I'm not sure what you mean 😅